Failure on x86_64-darwin [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34623&attempt_id=e24dd1bb-a0a9-4193-95bb-d18113c8ee62)

<details><summary>Partial log (click to expand)</summary><p>

```
clang++ -O3 -fomit-frame-pointer -Wall -fsigned-char -DSTANDALONE -I. -Ibot -I../enet/include -c -o stream-standalone.o stream.cpp
clang++ -O3 -fomit-frame-pointer -Wall -fsigned-char -DSTANDALONE -I. -Ibot -I../enet/include -c -o tools-standalone.o tools.cpp
tools.cpp:800:14: fatal error: 'SDL_timer.h' file not found
    #include "SDL_timer.h"
             ^~~~~~~~~~~~~
1 error generated.
make: *** [Makefile:172: tools-standalone.o] Error 1
make: Leaving directory '/private/tmp/nix-build-assaultcube-01052017.drv-0/source/source/src'
builder for '/nix/store/yxqkdfbinqfcmqpd2prvpbv7qi6ffv3d-assaultcube-01052017.drv' failed with exit code 2
error: build of '/nix/store/yxqkdfbinqfcmqpd2prvpbv7qi6ffv3d-assaultcube-01052017.drv' failed
```
</p></details>

--- a/pkgs/games/assaultcube/default.nix
+++ b/pkgs/games/assaultcube/default.nix
@@ -15,7 +15,7 @@ stdenv.mkDerivation rec {
     description = "Fast and fun first-person-shooter based on the Cube fps";
     homepage = http://assault.cubers.net;
     maintainers = [ maintainers.genesis ];
-    platforms = platforms.linux ++ platforms.darwin;
+    platforms = platforms.linux; # should work on darwin with a little effort.
     license = stdenv.lib.licenses.zlib;
   };
 

